assets/src/domain/eventEditor/services/apollo/mutations/datetimes/useOnCreateDatetime.ts

```diff
@@ -9,34 +11,37 @@ const useOnCreateDatetime = (): DatetimeMutationCallbackFn => {
 
 	const updateDatetimeCache = useUpdateDatetimeCache();
 
-	const onCreateDatetime = ({ proxy, datetimes, datetime, tickets }: DatetimeMutationCallbackFnArgs): void => {
-		if (datetime.id) {
-			const { nodes = [] } = datetimes;
-			const datetimeIn = getGuids(nodes).sort();
-			const { id: datetimeId } = datetime;
-
-			// Update tickets cache for the changed datetimes,
-			// to avoid refetching of tickets.
-			updateTicketCache({ proxy, datetimeIn, datetimeId, action: 'add' });
-
-			updateRelations({
-				entity: 'datetimes',
-				entityId: datetimeId,
-				relation: 'tickets',
-				relationIds: tickets,
-			});
-			tickets.forEach((entityId: string) => {
-				addRelation({
-					entity: 'tickets',
-					entityId,
-					relation: 'datetimes',
-					relationId: datetimeId,
+	const onCreateDatetime = useCallback(
```

Comment:
I also added useCallback :)
in this PR: https://github.com/eventespresso/event-espresso-core/pull/2862